tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -6255,7 +6255,10 @@ <h2>Show similarity to selected datapoint</h2>
             modelNum < extraOutputs.extra.length;
             modelNum++
           ) {
-            if ('attributions' in extraOutputs.extra[modelNum]) {
+            if (
+              extraOutputs.extra[modelNum] != null &&
```

Comment:
I don't remember the exact code path for this right now. Can there be a case where one model returns extraOutputs and other doesn't? In that case do we still create the extraOutputs.extra array of the correct size (2) and fill one with null?

Reply:
yes that is the behavior